verilog/formatting/align.cc

```diff
@@ -106,6 +106,34 @@ static bool IgnoreWithinPortDeclarationPartitionGroup(
   return false;
 }
 
+static bool IgnoreWithinStructUnionMemberPartitionGroup(
+    const TokenPartitionTree& partition) {
+  const auto& uwline = partition.Value();
+  const auto token_range = uwline.TokensRange();
+  CHECK(!token_range.empty());
+  // ignore lines containing only comments
+  if (TokensAreAllCommentsOrAttributes(token_range)) {
+    return true;
+  }
+
+  // ignore partitions belonging to preprocessing directives
+  if (IsPreprocessorKeyword(
+          verilog_tokentype(token_range.front().TokenEnum()))) {
+    return true;
```

Comment:
We should probably also have a test that covers this branch. Could you make a separate small pull request (as this is already merged) ?

Reply:
https://github.com/google/verible/pull/855